libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -289,19 +289,44 @@ NS_SWIFT_NAME(SalesforceManager)
 - (nonnull NSString *)devInfoTitleString;
 
 /**
- * Returns the ScreenLockManager instance.
+ * Returns The ScreenLockManager instance.
  *
- * @return the Screen Lock Manager
+ * @return The Screen Lock Manager.
  */
 - (id <SFScreenLockManager>)screenLockManager;
 
 /**
- * Returns the BiometricAuthenticationManager instance.
+ * Returns The BiometricAuthenticationManager instance.
  *
- * @return the Biometric Authentication Manager
+ * @return The Biometric Authentication Manager.
  */
 - (id <SFBiometricAuthenticationManager>)biometricAuthenticationManager;
 
+/**
+ * Creates the NativeLoginManager instance.
+ *
+ * @param consumerKey The Connected App consumer key.
+ * @param callbackUrl The Connected App redirect URI.
+ * @param communityUrl The login url for native login
+ * @param nativeLoginViewController The UI instance that will be presented to the user and
+ *  is responsible for using the returned Native Login Manager to initiate the login process.
+ * @param scene Optional UIScene to enable multi-window support.
+ *
+ * @return The Native Login Manager.
+ */
+- (id <SFNativeLoginManager>)useNativeLoginWithConsumerKey:(nonnull NSString *)consumerKey
```

Comment:
So an app can setup a native login that uses a different consumer key from the one in bootconfig?
Are we storing the consumer key alongside the refresh token or are we refreshing using the one from bootconfig?

Reply:
[I think it has always been stored with the user](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/d95c81823341a9076ae549fc3ea5b1d8e094b370/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m#L273).

Reply:
I guess I forgot - it's also the case on [Android](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/AuthenticatorService.java#L236).